objective_epoch.py

```diff
@@ -0,0 +1,220 @@
+from mlmicrophysics.models import DenseNeuralNetwork
+from mlmicrophysics.data import subset_data_files_by_date, assemble_data_files
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler, MinMaxScaler, MaxAbsScaler, RobustScaler, OneHotEncoder
+from sklearn.metrics import confusion_matrix, accuracy_score, mean_absolute_error, mean_squared_error
+from mlmicrophysics.metrics import heidke_skill_score, peirce_skill_score, hellinger_distance, root_mean_squared_error, r2_corr
+import argparse
+import yaml
+from os.path import join, exists
+import os
+from datetime import datetime
+import logging
+from keras.losses import huber_loss
+
+import optuna
+from aimlutils.hyper_opt.utils import trial_suggest_loader
+from aimlutils.hyper_opt.base_objective import *
+
+
+logger = logging.getLogger(__name__)
+
+scalers = {"MinMaxScaler": MinMaxScaler,
+           "MaxAbsScaler": MaxAbsScaler,
+           "StandardScaler": StandardScaler,
+           "RobustScaler": RobustScaler}
+
+class_metrics = {"accuracy": accuracy_score,
+                 "heidke": heidke_skill_score,
+                 "peirce": peirce_skill_score}
```

Comment:
For future evaluations, it would be helpful to include the probability of detection, probability of false detection, false alarm ratio, and frequency bias as class_metrics. The current scores are all good summaries but don't speak to the tradeoff between misses and false alarms.

Reply:
I think I've updated this by adding sklearn confusion matrix. Let me know if that doesn't encompass the additional metrics.